app/controllers/appointment_controller.rb

```diff
@@ -0,0 +1,20 @@
+require 'date'
+class AppointmentController < ApplicationController
+    def create_appointment
+        doa = DateTime.parse("#{params[:date]}T#{params[:time]}")
+        @appointment = Appointment.create(
+          date_of_appointment: doa,
+          no_of_session: params[:no_of_session],
+          doctor_id: params[:doctor_id],
+          patient_id: params[:patient_id],
```

Comment:
Why is there patient_id, the patient_id should be the logged in user, which is `current_user.id`